static/js/telescope_availability_app.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
I don't see the need to have this vue file and telescope_availability_app.js. Just merge this into the .js file and keep the component separate.